commit 1978d0a12d2cb4886c5612d6a50b145f409d815c
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Wed Oct 26 22:42:47 2016 +0200

    Switch from KF5Gpgmepp to Gpgmepp
    
    Gpgmepp is part of GpgME since 1.7.0.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index edf3e79..6993d2e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ trojita_plugin_option(WITH_QTKEYCHAIN_PLUGIN "Build Qtkeychain password plugin"
 trojita_find_package(Git "" "" "" "")
 
 trojita_find_package(Mimetic "" "http://www.codesink.org/mimetic_mime_library.html" "C++ MIME Library" "Required for client-side MIME parsing" WITH_MIMETIC)
-trojita_find_package(KF5Gpgmepp "" "https://quickgit.kde.org/?p=gpgmepp.git" "C++ bindings for gpgme" "Needed for encrypted/signed e-mails" WITH_GPGMEPP)
+trojita_find_package(QGpgme 1.8.0 "http://www.gnupg.org" "Qt5/C++ bindings for GpgME" "Needed for encrypted/signed e-mails" WITH_GPGMEPP)
 trojita_option(WITH_CRYPTO_MESSAGES "Enable support for encrypted messages" AUTO "WITH_MIMETIC;WITH_GPGMEPP")
 
 if(WIN32)
@@ -615,11 +615,7 @@ if(WITH_MIMETIC)
 endif()
 if(WITH_CRYPTO_MESSAGES)
   if(WITH_GPGMEPP)
-    if (WIN32)
-      target_link_libraries(Cryptography KF5::Gpgmepp KF5::QGpgme)
-    else()
-      target_link_libraries(Cryptography KF5::Gpgmepp-pthread KF5::QGpgme)
-    endif()
+    target_link_libraries(Cryptography QGpgme)
   endif()
 endif()
 
